implementers-guide/sections/05-nomenclature.adoc

```diff
@@ -0,0 +1,19 @@
+
+
+== Nomenclature
+
+////
```

Comment:
I think we should give an actual example. Can you find one on this page? https://www.itu.int/en/ITU-T/studygroups/com15/Pages/impguides.aspx

Reply:
I couldn't find a nomeclature example in the provided link. So, I put the example shown in the original template doc.